app/javascript/components/things.js

```diff
@@ -0,0 +1,37 @@
+import React, { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+// import { bindActionCreators } from 'redux';
```

Comment:
- [ ] <b>REQUIRED:</b> To make sure that your codebase is always clean, remember to remove commented-codes.